static/src/javascripts/projects/commercial/modules/dfp/on-slot-render.ts

```diff
@@ -0,0 +1,100 @@
+import { adSizes } from '@guardian/commercial-core';
+import type { AdSizeString } from '@guardian/commercial-core';
+import { once } from 'lodash-es';
+import config from '../../../../lib/config';
+import mediator from '../../../../lib/mediator';
+import reportError from '../../../../lib/report-error';
+import { markTime } from '../../../../lib/user-timing';
+import { dfpEnv } from './dfp-env';
+import { emptyAdvert } from './empty-advert';
+import { getAdvertById } from './get-advert-by-id';
+import { renderAdvert } from './render-advert';
+
+const recordFirstAdRendered = once(() => {
+	markTime('Commercial: First Ad Rendered');
```

Comment:
Isn’t this a duplicate of what we do with the `EventTimer`?

Reply:
Good spot.

We seem to use a mix of `markTime` and `EventTImer` across the commercial codebase.

So I think consolidating this is outside of this PR but I will raise a ticket for it.